swap/src/bin/swap.rs

```diff
@@ -268,6 +273,86 @@ async fn main() -> Result<()> {
 
             cli::refund(swap_id, Arc::new(bitcoin_wallet), db, force).await??;
         }
+        Command::ListMakers {
+            rendezvous_node_peer_id,
+            rendezvous_node_addr,
+            namespace,
+            tor_socks5_port,
+        } => {
+            // TODO: Fix tracing to be able to operate without swap ID
+            cli::tracing::init(debug, json, data_dir.join("logs"), Uuid::new_v4())?;
+            let seed = Seed::from_file_or_generate(data_dir.as_path())
+                .context("Failed to read in seed file")?;
+
+            let identity = seed.derive_libp2p_identity();
+
+            // TODO: Compose behaviour that allows to requests Quotes as well
```

Comment:
Very interesting idea, I like it.

Reply:
I think this one was already outdated when I pushed it. I think the behaviour that composes `Rendezvous` and `Quote` together is fine for now.